### ISSUE TITLE
Unable to start after updating ffmpeg

### ISSUE THREAD

**Issue by fj-02** [iEvgeny/cctv-viewer#49]
After updating my system I noticed the program doesn't start anymore.
This is what is being printed in the terminal:
`cctv-viewer: error while loading shared libraries: libavformat.so.59: cannot open shared object file: No such file or directory`

After some digging it looks like ffmpeg was updated and the shared library was updated and changed from libavformat.so.59 to libavformat.so.60

**Comment by iEvgeny** [comment]
Need more info: OS version, arch, installation method.

**Comment by fj-02** [comment]
Arch Linux, 6.2.2-zen2-1-zen, installed through AUR using yay

**Comment by iEvgeny** [comment]
In this case, in order to solve this problem, you need to contact the package maintainer in the AUR: https://aur.archlinux.org/packages/cctv-viewer-git

**Comment by fj-02** [comment]
Attempting to reinstall the package causes an error (something about qt6), and I tried the patch that is on the AUR (https://aur.archlinux.org/packages/cctv-viewer-git#comment-884680) after manually compiling and installing the package resulted in the program working. With hindsight I should've first checked the AUR, oh well.

Thanks for your time and for making this program, sorry if I've bothered you.
